app/Http/Controllers/Auth/LoginController.php

```diff
@@ -25,7 +25,7 @@ class LoginController extends Controller
      *
      * @var string
      */
-    protected $redirectTo = '/home';
+    protected $redirectTo = '/admin/dashboard';
```

Comment:
Hardcoding URIs ain't cool. Let's just use a named route. Say "home-dashboard" (which will need to be created)

Reply:
I didn't do the logic yet, just set up the routes/views for login/register/pass forget.

Reply:
Gotcha. Adjust the redirect to be $redirectTo = 'Name of the User Dashboard route'. So that we know in the future what to put there.